rslib/src/media/check.rs

```diff
@@ -869,4 +870,33 @@ Unused: unused.jpg
 
         Ok(())
     }
+
+    /// Regression test for https://github.com/ankitects/anki/issues/1909
+    #[test]
+    fn long_filename_rename_not_reported_as_unused() -> Result<()> {
+        let (_dir, mgr, mut col) = common_setup()?;
+
+        // 65 Greek α characters (2 bytes each) + ".mp3" = 134 bytes
+        let long_filename = format!("{}.mp3", "α".repeat(65));
+        assert!(long_filename.len() > 120);
```

Comment:
I'd use a sequence of "a" as suggested by @GithubAnon0000 and remove this assert.